Program.cs

```diff
@@ -65,6 +66,15 @@ private static void Main(string[] args)
 
             try
             {
+                var vsInstances = MSBuildLocator.QueryVisualStudioInstances();
+                var vs2017 = vsInstances.FirstOrDefault(x => x.Version.Major == 15);
+                var vs2019 = vsInstances.FirstOrDefault(x => x.Version.Major == 16);
+
+                VisualStudioInstance usedInstance = vs2019 ?? vs2017;
+                if (usedInstance == null)
```

Comment:
VS does not have to be running. I believe word "instance" is now used because same version of VS can be installed in parallel in different locations. In any case, I've just verified this manually.

![image](https://user-images.githubusercontent.com/80263149/148742920-9e1e8511-7ae0-412e-ad83-c3c75850f7b1.png)

Looking at `MSBuild.Locator` source code, it uses a native dll `Microsoft.VisualStudio.Setup.Configuration.Native.dll` which enumerates installations of Visual Studio, checks whether the installation was completed and whether it contains MSBuild.

Further refernces
https://docs.microsoft.com/en-us/visualstudio/extensibility/locating-visual-studio?view=vs-2022
https://github.com/microsoft/MSBuildLocator/blob/d83904bff187ce8245f430b93e8b5fbfefb6beef/src/MSBuildLocator/VisualStudioLocationHelper.cs